fmn/tasks.py

```diff
@@ -138,11 +138,13 @@ def run(self, message):
             message (dict): A fedmsg to find recipients for.
         """
         _log.debug('Determining recipients for message "%r"', message)
-
         topic, message_body = message['topic'], message['body']
-        if '.fmn.' in topic:
-            openid = message_body['msg']['openid']
-            fmn_lib.update_preferences(openid, self.user_preferences)
+
+        # We send a fake message with this topic as a broadcast to all workers in order for them
+        # to refresh their caches, so if this message is a cache refresh notification stop early.
+        if topic == 'fmn.internal.refresh_cache':
```

Comment:
This might also make sense to turn into a varaible that can be imported and used.